packages/react-storage/src/components/StorageBrowser/context/actions/__tests__/listLocationItemsAction.spec.ts

```diff
@@ -38,11 +43,13 @@ describe('listLocationItemsAction', () => {
       .mockResolvedValueOnce({
         // @ts-expect-error - JS union interfaces casue type issues
         items: generateMockItems(100),
+        excludedSubpaths: generateMockSubpaths(10),
         nextToken: 'first',
       })
       .mockResolvedValueOnce({
         // @ts-expect-error - JS union interfaces casue type issues
         items: generateMockItems(100),
+        excludedSubpaths: generateMockSubpaths(10),
```

Comment:
We should make sure that if we are including `excludedSubpaths` here that we are using `delimiter` in the input to `listLocationItemsAction`

Reply:
Fixed